modules/tinymce/src/themes/silver/test/ts/browser/editor/scrolling/EditorInScrollingContainerTest.ts

```diff
@@ -886,10 +880,23 @@ describe('browser.tinymce.themes.silver.editor.scrolling.EditorInScrollingContai
           const header = getEditorUi(editor, ui.editor.stickyHeader);
           const scroller = getAncestorUi(editor, ui.ancestors.scrollingWrapper);
 
-          scroller.dom.scrollTo(0, 0);
+          scroller.dom.scrollTo(0, heights.largeBanner);
+          await pWaitUntilAppears(header);
+
+          scroller.dom.scrollTo(0, heights.largeBanner - heights.scroller + 20);
+          await pWaitUntilAppears(header);
+
+          scroller.dom.scrollTo(0, heights.largeBanner - heights.scroller);
           await pWaitUntilDisappears(header);
 
-          scroller.dom.scrollTo(0, 10000);
+          scroller.dom.scrollTo(0, heights.largeBanner);
+
+          await pWaitUntilAppears(header);
+
+          scroller.dom.scrollTo(0, heights.editor + heights.largeBanner - header.dom.offsetHeight);
+          await pWaitUntilAppears(header);
+
+          scroller.dom.scrollTo(0, heights.editor + heights.largeBanner - header.dom.offsetHeight + 20);
```

Comment:
Is the `+ 20` of specific importance or could it be `+ 10`? More just wondering if it should be a variable so it isn't a magic number is all

Reply:
Moved to a variable, that's the value of the paragraph's margin.